ddtrace/internal/datadog/profiling/dd_wrapper/include/exporter.hpp

```diff
@@ -208,7 +208,7 @@ class Profile {
   void push_frame(std::string_view name,     // for ddog_prof_Function
                   std::string_view filename, // for ddog_prof_Function
                   uint64_t address,          // for ddog_prof_Location
-                  int64_t line               // for ddog_prof_Line
+                  int64_t line               // for ddog_prof_Location
```

Comment:
Does a `ddog_prof_Location` have both an `address` and `line`?

Reply:
Yeah, this change was made around version 5 when the structs for the API were flattened.